ios/AutomaticProperties.swift

```diff
@@ -5,7 +5,7 @@ class AutomaticProperties {
     static var peopleProperties: Dictionary<String, MixpanelType> = [:];
     
     static func setAutomaticProperties(_ properties: [String: Any]) {
-        for (key,value) in properties {
+        for (key,value) in properties ?? [:]  {
```

Comment:
This is in line with how we process the same `properties` object elsewhere: https://github.com/mixpanel/mixpanel-react-native/blob/master/ios/MixpanelTypeHandler.swift#L77